### DISCUSSION TITLE
Adding data and aircraft to FlightGear

### DISCUSSION THREAD

**Discussion by Jsousa576** [JSBSim-Team/jsbsim#1052] [General]
I'm doing my course completion work in mechanical engineering and I'm part of an aerodesign team, my goal is to add my team's plane to flightgear using JSBSim, and feed it with data from my aircraft, using python, Could anyone help me? I have no idea how to work with JSBSim.

**Comment by seanmcleod** [comment]
Have you read the readme - https://github.com/JSBSim-Team/jsbsim/blob/master/README.md and the JSBSim Reference Manual that it links to?

**Reply by Jsousa576** [reply]
I briefly read the README and the manual, but I don't even know the basics of using JSBSim, I don't even know how to initialize it in FlightGear.

**Comment by seanmcleod** [comment]
You mentioned in the duplicate discussion that you created, since deleted, that you couldn't successfully run FlightGear and fly an aircraft.

I presume you've looked at FlightGear' manual which details how to start FlightGear, select an aircraft, the controls to fly etc.?

https://flightgear.sourceforge.net/manual/2020.3/en/getstart-en.html
